FFCDemoPaymentService/Startup.cs

```diff
@@ -29,7 +29,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddDbContext<ApplicationDbContext>(options =>
-                options.UseNpgsql(Configuration.GetConnectionString("DefaultConnection")));
+                options.UseNpgsql(Configuration.GetConnectionString("DefaultConnection"), o => o.SetPostgresVersion(9, 6)));
```

Comment:
I'm not familiar with the migration issue that's been mentioned.  Is version 9.6 the instance of Postgres we have in Azure and is there an issue we have running that version without explicitly saying so?

Reply:
It resolves an issue in the migrations where it was using unsupported SQL syntax in version Postgres 9.6 as described in thos issie https://github.com/npgsql/efcore.pg/issues/1039. I've updated the PR description to reference the other card it resolves.